components/NewPersonView/PreviewBanner.tsx

```diff
@@ -20,24 +20,16 @@ const PreviewBanner = ({
         className="lbh-page-announcement lbh-page-announcement--orange"
       >
         <h4 className="lbh-page-announcement__title">
-          Try the faster, brand new resident view
+          This view will soon be retired
         </h4>
         <div className="lbh-page-announcement__content">
           <p className="lbh-body-s">
             You can record much more information about a resident, see a
-            chronology of their workflows and browse case notes faster.
+            chronology of their workflows and browse case notes faster using the
+            new resident view.
```

Comment:
Is it worth linking the new resident view here?

Reply:
it is linked right below